apps/core/.env.example

```
@@ -9,3 +9,6 @@ SOROBAN_RPC_URL="https://soroban-testnet.stellar.org"
 TOKEN_FACTORY_WASM_HASH=""
 PARTICIPATION_TOKEN_WASM_HASH=""
 VAULT_WASM_HASH=""
+
+BACKOFFICE_API_KEY= "26fefb1fcf7709502edd4ac954572a89ce57273c33545347583697520091f6c3"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)